packages/solid-crs-manage/lib/features/loan/loan.context.ts

```diff
@@ -0,0 +1,20 @@
+import { LoanRequest } from '@netwerk-digitaal-erfgoed/solid-crs-core';
+
+export interface LoanContext {
+  /**
+   * All `LoanRequest`s retrieved
+   */
+  loanRequests?: LoanRequest[];
+  /**
+   * The loan request currently being viewed in the detail page
+   */
+  loanRequest?: LoanRequest;
```

Comment:
I wonder whether these optional attributes are actually needed. just a thought.

Reply:
Probably not but I can imagine that in some places it won't autofill / show documentation if left out.
Also, I don't think empty interfaces are 'allowed' / good practice.

Reply:
alright let's keep it in